rest-api/etl/export_etl_results.sh

```diff
@@ -13,18 +13,26 @@ while true; do
   esac
 done
 
-if [ -z "${ACCOUNT}" ] || [ -z "${PROJECT}" ] || [ -z "${DIRECTORY}" ]
+if [ -z "${DIRECTORY}" ]
 then
   echo "Usage: $USAGE"
   exit 1
 fi
+PROJECT_AND_ACCOUNT=
+if [ "${PROJECT}" ]
+then
+  PROJECT_AND_ACCOUNT="--project ${PROJECT} --account ${ACCOUNT}"
```

Comment:
Both project and account are required, right? Why not treat them as two arguments below?

Reply:
No -- you can run this locally without specifying project or account, and it will hit the local server (and dump files to the storage dir on disk.)

I updated the usage to indicate this.